ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -106,11 +106,14 @@ Item {
         anchors.topMargin: 8
         anchors.bottomMargin: Style.current.halfPadding
         anchors.horizontalCenter: parent.horizontalCenter
+        enabled: !root.communityData.amIBanned
 
-        visible: !communityData.joined
+        visible: !communityData.joined || root.communityData.amIBanned
```

Comment:
Additional, in the backend, we should switch `joined` to `false` when we get the signal that we are banned.
I'd still keep this here as a security, but changing the `joined` flag would give more certainty.

Reply:
Is there any ticket about this ?